spec/features/student_user_registers_for_courses_spec.rb

```diff
@@ -50,26 +50,25 @@
     visit courses_path
     click_button "Register for #{date.second.strftime('%B')} Classes"
 
-    expect(page).to have_button("Register for #{date.second.strftime('%B')} Classes")
-    expect(page).to have_button("Register for #{date2.second.strftime('%B')} Classes")
-
-    select "Follow", from: "registration_holder[first_role]", match: :first
-    select "Lead", from: "registration_holder[second_role]", match: :first
-    select "Lead", from: "registration_holder[third_role]", match: :first
-    select "Follow", from: "registration_holder[fourth_role]", match: :first
+    all('input[type=checkbox]').each do |checkbox|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Pass `&:click` as an argument to `each` instead of a block.